src/index.html

```diff
@@ -1,14 +1,38 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
-    <meta charset="UTF-8">
-    <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <meta charset="UTF-8" >
+    <meta
+      name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+    >
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" >
     <title>Moyo header</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="./style.css" >
+    <link rel="preconnect" href="https://fonts.googleapis.com" >
+    <link rel="preconnect" href="https://fonts.gstatic.com" >
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap"
+      rel="stylesheet"
+    >
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <img class="img__logo"
+      src="images/logo.png" alt="moyo-logo">
+      <nav class="nav">
+        <a class="nav__link is-active" href="#apple">Apple</a><!--
+        --><a class="nav__link" href="#Samsung">Samsung</a><!--
+        --><a class="nav__link" href="#Smartphones">Smartphones</a><!--
+        --><a class="nav__link"
+          href="#Laptops"
+          data-qa="hover"
+        >Laptops & Computers</a><!--
```

Comment:
```suggestion
        <a class="nav__link is-active" href="#apple">Apple</a>
        <a class="nav__link" href="#Samsung">Samsung</a>
        <a class="nav__link" href="#Smartphones">Smartphones</a>
        <a 
          class="nav__link"
          href="#Laptops"
          data-qa="hover"
        >
          Laptops & Computers
        </a>
          
```
why do you need this comments?
remove also from other places

Reply:
I used this to remove extra spaces.